spec/feature/remember_me_spec.rb

```diff
@@ -52,6 +52,20 @@
         visit_change_number_page
         expect(page).to have_text(I18n.t("mfa.phone.code.redo_description_preamble"))
       end
+
+      context "the user has already re-done MFA" do
+        before do
+          visit_change_email_page
+          redo_mfa
+          expect(page).to have_text(I18n.t("devise.registrations.edit.heading_email"))
+          visit "/account"
```

Comment:
Should we use the path helpers over the hard coded path? Just in case we change it?